bin/reth/src/node/mod.rs

```diff
@@ -89,8 +90,9 @@ impl Command {
         let config: Config = confy::load_path(&self.config).unwrap_or_default();
         info!("reth {} starting", crate_version!());
 
-        let new_limit = raise_fd_limit().expect("Failed to raise the fd limit of the binary");
-        info!("Raising the fd limit of the binary to {}", new_limit);
+        // Raise the fd limit of the process.
+        // Does not do anything on windows.
+        raise_fd_limit().ok_or_else(|| eyre!("Failed to raise the fd limit"))?;
```

Comment:
use `[wrap_err](https://docs.rs/eyre/latest/eyre/trait.WrapErr.html#tymethod.wrap_err)` instead

Reply:
wrap_err is not implemented for Option. The docs recommend using ok_or_else. 
https://docs.rs/eyre/latest/eyre/index.html#context-and-option

Reply:
ah sorry for the confusion then. somehow thought this returned an err.


looking at this https://github.com/paritytech/fdlimit/blob/master/src/lib.rs

we can just throw away the Option because the `None` variant is for `windows` only, so we don't need to handle this case.